### ISSUE TITLE
⌘A should select all of the content

### ISSUE THREAD

**Issue by aaronjorbin** [WordPress/gutenberg#2292]
Right now,  ⌘A only selects the current block of content.  This means that it's impossible in the Visual editor to select all of the content only using the keyboard.

**Comment by youknowriad** [comment]
If a block is selected yet (an input, a textarea, an editable),  ⌘A will select only this area but if you focus something else it will select the whole content. That's the tradeoff we're making right now.

**Comment by jasmussen** [comment]
We discussed this in Slack briefly, and I misunderstood the question being asked. And so this ticket is actually expected behavior per discussion in #3. 

We also discussed copy and paste behavior, which as it turns out is _semi functional_ but needing a lot of polish. Here's a video also showing the ⌘ A behavior on the blocks. But I'm closing this one for now, and will reopen a separate ticket for copy/paste improvements.
